src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonName } from '../PersonName';
+import { Sex } from '../../types/Sex';
+
+type PersonLinkProps = {
+  person: Person & {
+    motherNameLink: string | null;
+    fatherNameLink: string | null;
+  };
+};
+
+export const PersonLink: React.FC<PersonLinkProps> = ({
+  person: {
+    name,
+    slug,
+    sex,
+    born,
+    died,
+    motherName,
+    fatherName,
+    motherNameLink,
+    fatherNameLink,
+  },
+}) => {
+  const { personId } = useParams();
```

Comment:
```suggestion
  person,
}) => {
  const {
    name,
    slug,
    sex,
    born,
    died,
    mother,
    father,
  } = person;
  const { personId } = useParams();
```
- Avoid nested destructing, you can do it below
- If you store whole father/mother objects you can get slug and name from it and you don't need so many fields.